doc/source/user_guide/scale.rst

```diff
@@ -18,6 +18,23 @@ tool for all situations. If you're working with very large datasets and a tool
 like PostgreSQL fits your needs, then you should probably be using that.
 Assuming you want or need the expressiveness and power of pandas, let's carry on.
 
+.. ipython:: python
+
+   import pandas as pd
+   import numpy as np
+
+.. ipython:: python
+   :suppress:
+
+   from pandas._testing import _make_timeseries
```

Comment:
Not sure whether private functions should be used in public docs

Reply:
I agree. Removed!